lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
Cancelling a request is more of a limit action that simply marks a *request* as cancelled after the fact. This is only coming up because internally in Shopify we have a case where the fulfillment provider doesn't know initially that items can or can not be fulfilled.

Cancelling a fulfillment actually can be done already [here](https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/resources/fulfillment.rb#L9).